types/chai/index.d.ts

```diff
@@ -762,6 +763,16 @@ declare namespace Chai {
          */
         isNotNumber<T>(value: T, message?: string): void;
 
+        /**
+         * Assert that value is a finite number.
```

Comment:
For consistency with other methods, please replace word `Assert` with `Asserts`.